README.md

```diff
@@ -1,5 +1,8 @@
 # Wheatley
-A bot for Ringing Room that can fill in any set of bells to increase the scope of potential practices.
+[![PyPI version](https://badge.fury.io/py/wheatley.svg)](https://badge.fury.io/py/wheatley)
+![Tests and Linting](https://github.com/Kneasle/wheatley/workflows/Tests%20and%20Linting/badge.svg)
```

Comment:
I assume this just show the result of the latest run on any branch/pull request?

Possibly it could be restricted to builds on master but any failures ought to be investigated and fixe anyway.

Reply:
This will show the result of the run _after_ the last PR to master - this _should_ always say 'passing', but it's more just to make people confident that we check things like code style and tests.

This doesn't stop the Lint & Tests from running on incoming PRs - the results just won't show on the badge.